components/rendering/src/markdown.rs

```diff
@@ -192,6 +194,19 @@ pub fn markdown_to_html(content: &str, context: &RenderContext) -> Result<Render
                 match event {
                     Event::Text(text) => {
                         // if we are in the middle of a code block
+                        if &context.config.highlight_theme == "css" {
+                            if let Some(ref mut css_highlighter) = css_highlighter {
+                                for line in text.lines() {
+                                    css_highlighter.parse_html_for_line(&line);
+                                }
+                                // swap out highlighter because of borrowing
+                                let sr_rs = &SYNTAX_SET.find_syntax_by_extension("rs").unwrap();
```

Comment:
Is this only highlighting rust..?

Reply:
Maybe the panic is because of that? It's trying to parse a different language than it is

Reply:
At this point i only need a unused syntax highlighter for swapping it out before calling finalize (We discussed that before). 

The highlighter which is used to do the actual highlighting is created in line 239 with the correct syntax set from the codeblock:
```
 match &SYNTAX_SET.find_syntax_by_extension(info) {
```